data/voracious.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "David Myers",
+  "url": "https://github.com/voracious",
+  "image": "https://github.com/voracious.png",
+  "why": "To find others that enjoy fitness and coding",
+  "weekly": ["Lift 2x", "Cycle 1x", "Wake up a little earlier than last week"],
```

Comment:
Welcome David! 
I'll merge this for now but consider having a quantitative measure of "waking up a little earlier"! 
Cheers!

Reply:
Thanks, @katalystkat! I'll come up with an approximate weekly target and open another PR.